### ISSUE TITLE
root fields: comment and acquisition time

### ISSUE THREAD

**Issue by smXplorer** [Photon-HDF5/photon-hdf5#20]
The two lone root fields "comment" and "acquisition_time" may benefit from the following renaming:

1) comment -> description

2) acquisition_time -> acquisition_duration

"comment" is fine, but from the files I have read so far, that comment consists in a brief description of the experiment that the file is supposed to contain the data of. Why not call it what it is then?

"acquisition_time" is ambiguous in English. It really means the time at which the acquisition was performed, not the duration of the measurement, which is what the content is.

So basically, the suggestion is to disambiguate those two fields.
